misk/web/tabs/web-actions/src/containers/WebActionsContainer.tsx

```diff
@@ -288,81 +256,64 @@ export const FilterWebActions = (props: IState & IDispatchProps) => {
     </div>
   )
 }
+const SkeletonText = () => (
+  <span className={Classes.SKELETON}>{"Lorem ipsum"}</span>
```

Comment:
hmm will this ever show up to devs?

Reply:
If computation takes too long, usually may flash prior to being filled in with the real data. Similar to the Skeleton table we already use in SampleTableComponent in Misk-Web/Palette

https://square.github.io/misk-web/examples/tabs/palette/demo/

Reply:
why not put no txt?

Reply:
A bit unintuitive at first glance but allows a way to mock the final UI when the data has loaded: https://blueprintjs.com/docs/#core/components/skeleton